skyvern/forge/agent.py

```diff
@@ -943,7 +945,6 @@ async def agent_step(
                         complete_action.task_id = task.task_id
                         complete_action.step_id = step.step_id
                         complete_action.step_order = step.order
-                        complete_action.action_order = len(detailed_agent_step_output.actions_and_results)
```

Comment:
what's the reason for removing this line?

Reply:
@LawyZheng we do want to keep this line correct?

Reply:
yes, we need to keep it.

Reply:
This and `InvalidUrl(SkyvernException):` are results of a bad merge

I'll revert those changes